.env.example

```diff
@@ -18,4 +18,9 @@ HEROKU_SLUG_COMMIT=<commit hash>
 SENTRY_AUTH_TOKEN=<sentry auth token>
 
 # Optional - feature toggles
-ENABLE_QUOTE_CART_API=false
+FEATURE_QUOTE_CART_API=<SE,DK,NO>
```

Comment:
Could it be an idea to have a value for enabling all markets? Or maybe that's when you should remove the feature flag instead :)

Reply:
haha yeah I guess so -- we can revisit that when we support say 10 markets

Reply:
I personally think it's preferable to be intentional when enabling features. Having empty mean all markets is risky and can lead to developer error.

I think we should rather remember to cleanup feature flags for things we know we will keep long term. For instance the quoteCartAPI flag should go away as soon as we are live on all markets.

Reply:
I agree, I was actually thinking that empty means disable the feature in all markets 😅 at least we need some way to disable a feature that is turned on by default.

Reply:
Sounds good! 🙌